S03-operators/repeat.t

```diff
@@ -44,6 +44,24 @@ is 'str' x Int, '', 'x with Int type object';
     isa-ok('a' x Inf, Failure, 'repeating with Inf is a Failure');
     isa-ok('a' x *, WhateverCode, 'repeating with * is a WhateverCode');
 }
+# RT #128035
+{
+    my $a;
+    lives-ok({ $a = 'a' x 1073741824 }, 'repeat count equal to the NQP limit works');
+    is($a.chars, 1073741824, 'correct result for count equal to the NQP limit');
+    throws-like(
+        { 'a' x 1073741825 },
+        X::AdHoc,
+        message => 'repeat count > 1073741824 arbitrarily unsupported...',
+        'repeat count greater than the NQP limit dies'
+    );
+    throws-like(
+        { 'a' x int.Range.max + 1 },
+        X::AdHoc,
+        message => 'repeat count > 1073741824 arbitrarily unsupported...',
+        'repeat count greater than the max size of an int dies because of the NQP limit'
+    );
```

Comment:
This last test fails on current Rakudo, because `int.Range.max + 1` evaluates to 0.